Sources/StreamChat/Models/ChatMessage.swift

```diff
@@ -168,7 +170,7 @@ public struct ChatMessage {
         currentUserReactions: @escaping () -> Set<ChatMessageReaction>,
         isSentByCurrentUser: Bool,
         pinDetails: MessagePinDetails?,
-        attachmentCounts: @escaping () -> [AttachmentType: Int],
+        attachmentCounts: [AttachmentType: Int],
```

Comment:
it seems `attachmentCounts` can be excluded from `init` params since we transformed it to computed property 👍